app/services/document_type.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-class DocumentTypeSchema
+class DocumentType
```

Comment:
I think there's a bit more work we can do here, as I'd like to try and maintain the convention that things in models/ support ActiveModel before moving these files in there. We did try this originally, but I don't think we followed ActiveModel conventions and that lead us to avoid it entirely. I was hoping to write another card to look at this again.

Reply:
ok I see - from my perspective I don't think it's worth making things support activemodel if they are immutable things which I'd suggest these are.

I understand the perspective of using ActiveModel makes it a model, but I don't think we should apply that to things that don't need the write aspects of it unnecessarily.

Reply:
Thanks for the extra discussion on this - hopefully we'll revisit another time...